xcconfigs/Prelude.xcconfig

```diff
@@ -15,3 +15,5 @@ MACOSX_DEPLOYMENT_TARGET = 10.13
 INFOPLIST_FILE = Supporting Files/Info.plist
 
 APPLICATION_EXTENSION_API_ONLY = YES
+
+#include? "../../../../Prelude-Optional-Custom.xcconfig"
```

Comment:
1. Can we make this somehow less brittle than moving up four `..`s, which is sort of coupling this change to Carthage implementation details?

2. How do we feel about the name of this file specifically? I wanted it to be clear to people who might see this that they can safely ignore it unless they know they need something like this.

Reply:
Gonna at least add a comment here explaining the usage of this override.